tests/test_channel_properties.py

```diff
@@ -0,0 +1,197 @@
+
+    @pytest.mark.parametrize("seed", [generate_random_seed()])
+    def test_enum_property(self, sim_ts_power_device, seed):
+        """Test for validating channel property of enum type."""
+        # Reset the pseudorandom number generator with seed.
+        random.seed(seed)
+
+        pwr_phys_chan = f"{sim_ts_power_device.name}/power"
+        voltage_setpoint = random.random() * 6.0
+        current_setpoint = random.random() * 3.0
+        output_enable = random.choice([True, False])
+
+        with nidaqmx.Task() as task:
+            pwr_channel = task.ai_channels.add_ai_power_chan(
+                pwr_phys_chan,
+                voltage_setpoint,
+                current_setpoint,
+                output_enable,
+                name_to_assign_to_channel="PowerChannel",
+            )
+
+            # Test property initial value.
+            assert (
+                pwr_channel.pwr_idle_output_behavior
+                == PowerIdleOutputBehavior.MAINTAIN_EXISTING_VALUE
+            )
+
+            # Test property setter and getter.
+            pwr_channel.pwr_idle_output_behavior = PowerIdleOutputBehavior.OUTPUT_DISABLED
+            assert pwr_channel.pwr_idle_output_behavior == PowerIdleOutputBehavior.OUTPUT_DISABLED
+
+            # Test property deleter.
+            del pwr_channel.pwr_idle_output_behavior
+            assert (
+                pwr_channel.pwr_idle_output_behavior
+                == PowerIdleOutputBehavior.MAINTAIN_EXISTING_VALUE
+            )
+
+    @pytest.mark.parametrize("seed", [generate_random_seed()])
+    def test_float_property(self, any_x_series_device, seed):
+        """Test for validating channel property of float type."""
+        # Reset the pseudorandom number generator with seed.
+        random.seed(seed)
+
+        ai_phys_chan = random.choice(any_x_series_device.ai_physical_chans).name
+
+        with nidaqmx.Task() as task:
+            ai_channel = task.ai_channels.add_ai_rtd_chan(
+                ai_phys_chan,
+                name_to_assign_to_channel="RTDChannel",
+                min_val=28.0,
+                max_val=120.0,
+                units=TemperatureUnits.K,
+                rtd_type=RTDType.PT_3750,
+                resistance_config=ResistanceConfiguration.TWO_WIRE,
+                current_excit_source=ExcitationSource.EXTERNAL,
+                current_excit_val=0.0025,
+                r_0=100.0,
+            )
+
+            # Test property initial value.
+            assert ai_channel.ai_rtd_a == 0.00381
+
+            # Test property setter and getter.
+            value_to_test = random.random()
+            ai_channel.ai_rtd_a = value_to_test
+            assert ai_channel.ai_rtd_a == value_to_test
+
+            # Test property deleter.
+            del ai_channel.ai_rtd_a
+            assert ai_channel.ai_rtd_a == 0.00381
+
+    @pytest.mark.parametrize("seed", [generate_random_seed()])
+    def test_string_property(self, any_x_series_device, seed):
```

Comment:
you're not using the seed in this function, so I would just remove it. Same is true for most of these

Reply:
Removed unnecessary usage of seed in the tests